cloud/aws/templates/aws_oidc/bin/setup.py

```diff
@@ -75,3 +84,33 @@ def _maybe_set_secret_value(self, secret_name: str, documentation: str):
         else:
             print(f'Secret {secret_name} already has a value set.')
             print(f'You can check and update it in AWS console: {url}')
+
+    def _maybe_change_default_db_password(self):
+        """
+        Terraform creates database password secret with a random value and
+        creates database and ECS service that use that password. The problem
+        is that because password generated within terraform - its value is
+        stored in the Terraform state. To avoid exposing password in the state
+        this method regenerates password and updates database and server to use
+        the new password.
+        """
+        print()
+        print('Checking database password...')
+        app_prefix = self.config.app_prefix
+        secret_name = f'{app_prefix}-postgres_password'
```

Comment:
we have a lot of hardcoded strings for resource names here. Can you add a comment that these values must match names in .tf files and link them.

Reply:
Moved all resource names to separate file and added links to tf files. I would love to a test to ensure that they match whatever we produced in terraform state. But that will need to wait till we have a proper deployment test that actually produces that state.